packages/utils/src/lib/__snapshots__/report-to-md.integration.test.ts.snap

```diff
@@ -16,106 +16,108 @@ exports[`report-to-md > should contain all sections when using the fixture repor
-- 🟩 [Enforce comparing \`typeof\` expressions against valid strings](#enforce-comparing-typeof-expressions-against-valid-strings-eslint) (_ESLint_) - **0**
-- 🟥 [Require the use of \`===\` and \`!==\`](#require-the-use-of--and--eslint) (_ESLint_) - **1 warning**
-- 🟥 [Disallow missing \`key\` props in iterators/collection literals](#disallow-missing-key-props-in-iterators-collection-literals-eslint) (_ESLint_) - **1 warning**
-- 🟥 [Disallow missing props validation in a React component definition](#disallow-missing-props-validation-in-a-react-component-definition-eslint) (_ESLint_) - **6 warnings**
-- 🟩 [Disallow missing React when using JSX](#disallow-missing-react-when-using-jsx-eslint) (_ESLint_) - **0**
-- 🟩 [enforces the Rules of Hooks](#enforces-the-rules-of-hooks-eslint) (_ESLint_) - **0**
-- 🟥 [verifies the list of dependencies for Hooks like useEffect and similar](#verifies-the-list-of-dependencies-for-hooks-like-useeffect-and-similar-eslint) (_ESLint_) - **2 warnings**
+- 🟩 [Disallow assignment operators in conditional expressions](#disallow-assignment-operators-in-conditional-expressions-eslint) (_eslint_) - **0**
+- 🟩 [Disallow reassigning \`const\` variables](#disallow-reassigning-const-variables-eslint) (_eslint_) - **0**
+- 🟩 [Disallow the use of \`debugger\`](#disallow-the-use-of-debugger-eslint) (_eslint_) - **0**
+- 🟩 [Disallow invalid regular expression strings in \`RegExp\` constructors](#disallow-invalid-regular-expression-strings-in-regexp-constructors-eslint) (_eslint_) - **0**
+- 🟩 [Disallow the use of undeclared variables unless mentioned in \`/*global */\` comments](#disallow-the-use-of-undeclared-variables-unless-mentioned-in--global---comments-eslint) (_eslint_) - **0**
+- 🟩 [Disallow loops with a body that allows only one iteration](#disallow-loops-with-a-body-that-allows-only-one-iteration-eslint) (_eslint_) - **0**
+- 🟩 [Disallow negating the left operand of relational operators](#disallow-negating-the-left-operand-of-relational-operators-eslint) (_eslint_) - **0**
+- 🟩 [Disallow use of optional chaining in contexts where the \`undefined\` value is not allowed](#disallow-use-of-optional-chaining-in-contexts-where-the-undefined-value-is-not-allowed-eslint) (_eslint_) - **0**
+- 🟩 [Require calls to \`isNaN()\` when checking for \`NaN\`](#require-calls-to-isnan-when-checking-for-nan-eslint) (_eslint_) - **0**
+- 🟩 [Enforce comparing \`typeof\` expressions against valid strings](#enforce-comparing-typeof-expressions-against-valid-strings-eslint) (_eslint_) - **0**
+- 🟥 [Require the use of \`===\` and \`!==\`](#require-the-use-of--and--eslint) (_eslint_) - **1 warning**
+- 🟥 [Disallow missing \`key\` props in iterators/collection literals](#disallow-missing-key-props-in-iterators-collection-literals-eslint) (_eslint_) - **1 warning**
+- 🟥 [Disallow missing props validation in a React component definition](#disallow-missing-props-validation-in-a-react-component-definition-eslint) (_eslint_) - **6 warnings**
```

Comment:
The audits within a category seem to be sorted as they were defined in the config.

Maybe there was a misunderstanding - while I agree that the categories themselves should have an order in which they were defined, the audits and groups within them should be sorted based on weight > score > value > name in order not to mix the positive and negative results. What do you think?